packages/shared/src/saleor-version-compatibility-validator.ts

```diff
@@ -1,8 +1,5 @@
 const semver = require("semver");
 
-/**
- * TODO Extract to shared or even SDK
- */
 export class SaleorVersionCompatibilityValidator {
   constructor(private appRequiredVersion: string) {}
 
```

Comment:
What do you think about changing the error message to 
`Your Saleor version (${saleor version}) is not compatible with the App (${this.appRequiredVersion})`

Not these words exactly, but it will speed up debugging if the message sent to the User will contain both requirement and his current version. WDYT?

Reply:
Absolute bomb if error would emit "user friendly" message like "Update Saleor version", or "App is not yet compatible with new Saleor version you are using".
But it might require a bit of fun with semver and possible edge cases.

Reply:
Improved error message to include both versions